app/vmui/packages/vmui/src/utils/logs.ts

```diff
@@ -9,12 +9,12 @@ export const getStreamPairs = (value: string): string[] => {
   return pairs.filter(Boolean);
 };
 
-export const getHitsTimeParams = (period: TimeParams) => {
+export const getHitsTimeParams = (period: TimeParams, barsCount = LOGS_DEFAULT_BAR_COUNT) => {
```

Comment:
To avoid conflicts with PR #988, could you align this with the implementation from [https://github.com/VictoriaMetrics/VictoriaLogs/blob/08d08b9edaddda994a373ac516f7b67e6c90bfa4/app/vmui/packages/vmui/src/utils/logs.ts#L12-L22](https://github.com/VictoriaMetrics/VictoriaLogs/blob/08d08b9edaddda994a373ac516f7b67e6c90bfa4/app/vmui/packages/vmui/src/utils/logs.ts#L12-L22)